api/v1alpha2/zz_generated.conversion.go

```diff
@@ -464,6 +464,7 @@ func autoConvert_v1alpha3_ClusterStatus_To_v1alpha2_ClusterStatus(in *v1alpha3.C
 	out.InfrastructureReady = in.InfrastructureReady
 	out.ControlPlaneInitialized = in.ControlPlaneInitialized
 	// WARNING: in.ControlPlaneReady requires manual conversion: does not exist in peer-type
+	// WARNING: in.Conditions requires manual conversion: does not exist in peer-type
```

Comment:
We'll need some utilities to restore Conditions from the backup annotation, otherwise we'd risk of losing them during v1alpha3 -> v1alpha2 -> v1alpha3 conversion

Reply:
Actually, we only need to add `dst.Status.Conditions = restored.Status.Conditions` in the conversion.go file

Reply:
Done (implemented in `v1alpha/conversion.go`)